packages/client-core/src/systems/AvatarUISystem.tsx

```diff
@@ -296,6 +296,24 @@ const reactor = () => {
       getMutableState(PopupMenuState).menus[AvatarMenus.AvatarContext].set(none)
     }
   }, [])
+
+  return (
+    <>
+      <QueryReactor
+        Components={[AvatarComponent, TransformComponent, NetworkObjectComponent]}
```

Comment:
should add `Not(NetworkOwnerTagComponent)` to the list rather than `if (userId !== Engine.instance.store.userID)`

Reply:
oh nice I wasn't aware of that one. making the change